web/angular-wallet/src/app/layout/components/navbar/vertical/style-1/style-1.component.html

```diff
@@ -28,7 +28,7 @@
        [ngClass]="fuseConfig.layout.navbar.secondaryBackground">
 
     <div class="h3 username">{{ selectedAccount.accountRS }}</div>
-    <div class="h5 username">{{ getBalance() | burstAmount: 'short' }}</div>
+    <div class="h5 username">{{ getBalance() | burstAmount:'burst':true }}</div>
```

Comment:
I changed the pipe to accept planck also, furthermore there are two optional bools, which were 'short' and 'no-unit' before. In the IDE the semantics are shown...

Reply:
![image](https://user-images.githubusercontent.com/3920663/76156858-066cf980-60df-11ea-9f3a-95c83a2268b8.png)